From e588817e5d4228c16972ad3950b67a0f579fcc0f Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 8 Jan 2015 19:16:44 +0100
Subject: [PATCH] man: remove references to systemadm from systemctl man page

The tool is badly maintained and we shouldn't refence such old cruft.

(cherry picked from commit a03fe1a5d56be86fcc383090e191224590a1ab5d)
---
 man/systemctl.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/systemctl.xml b/man/systemctl.xml
index 7d643d796e..52fa803d58 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -1648,9 +1648,9 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
     <title>See Also</title>
     <para>
       <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>systemadm</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>loginctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>machinectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.resource-management</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
